### ISSUE TITLE
WE-MAPI_3020 Inductor 

### ISSUE THREAD

**Issue by unmaker64** [Opatusos/SNES_TST#2]
Hi is this the correct inductor? 

https://www.mouser.com/ProductDetail/Wurth-Elektronik/74438336068?qs=iC14BvcgY%2F2jjdn7Q0ci3w%3D%3D

Also wondering if this project will still be worked on, if you don't mind sharing. 

Thanks

**Comment by Opatusos** [comment]
Hi!

Is it from the dev branch? This inductor should be the correct one.
It is being worked on in theory, but I have not touched it since January. Hopefully I will find some time for it soon.

**Comment by unmaker64** [comment]
Thank you for the quick reply! 

I'm not sure how to find the most recent BOM within the dev branch tbh. 

I had a look at the PCB in Altium and could see the part is described as "WE-MAPI_3020". These WE-MAPI_3020 inductors range in value from 330 nH to 10 uH. I tried to determine the correct value by looking at the datasheet for the ADP2370 and found this:

![image](https://user-images.githubusercontent.com/110065276/181126330-836652ba-ca4e-457c-b8bf-bab9520e1277.png)

Using that formula I get an inductance value of 6.8 uH, assuming the Vout is 3.3v and the switching frequency is 1.22 MHz. I'm sort of guessing here so I wanted to check with you.

**Comment by unmaker64** [comment]
I'm glad to hear that you haven't abandoned it! I look forward to seeing your progress on this when you have the time. 

I'm not in a hurry to assemble a board anyway as I can't find the 10m04sae144c8g in stock anywhere. Which brings up another question I have: are other Altera MAX 10 FPGA's compatible? For example, a 10m08sae144c8g or 10M50SAE144I7G? A few of those have sold recently on ebay, which is the only place I've been able to find an FPGA.

**Comment by Opatusos** [comment]
You were correct, it is 6.8 uH, but for some reason it shows 0603 in the comment field instead of the value. I really like this ADP2370 combined with the LT3045 LDO. I have also made an LM7805 replacement board with these two. It delivers up to 800 mA and has reasonably low noise.

The FPGA should be compatible up to 10M25SA, but not with the 10M50SA. Not sure what is the difference.
I have attached a quick export of the BOM. But as I see most of the semiconductors are not in stock unfortunately.
[SNES TST Main.xlsx](https://github.com/Opatusos/SNES_TST/files/9203990/SNES.TST.Main.xlsx)

**Comment by unmaker64** [comment]
I really appreciate the BOM export. I was making my own by looking at the PCB in Altium. 

In the BOM I can see the ferrite beads you've chosen are 0402 in size yet the PCB footprint is 0603. I know, not a big deal as 0402 components can fit in a 0603 footprint. I'm just wondering if that was a small mistake. I had picked these just to match the 0603 footprint:

https://www.mouser.com/ProductDetail/810-MMZ1608R102ATD25

Yeah the parts shortage really sucks. I figured I'd order what's currently available before they go out of stock again. I've been able to find more than I expected actually. What I have left to find now are the:

1. FPGA
2. DAC 
3. clock 
4. TMUX1574DYYR 
5. IRLML6402 

A few of those are available on ebay/aliexpress. I might take a gamble on them if it comes to that. If/when I'm able to find the remaining parts then I'll order some PCBs and report back here. Thanks again for your time.

**Comment by Opatusos** [comment]
Yeah, I've changed the ferrite but did not change the details yet, as it was/is still WIP.

I have never tested this new version in the dev branch, so I'm not sure if it works at all. I still wanted to change the RGB lines from single-ended to differential. Right now I get GND interference if I connect the RGB signals to the Multiout connector.
